app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -349,7 +349,18 @@ function paginate(tag, params, requiresApproval) {
     searchHash.class_name = 'Article';
     const isHomePageFeed = window.location.pathname == '/';
     if (isHomePageFeed && userData()) {
-      searchHash.hidden_tags = userData().hidden_tags;
+      const hidden_tags = JSON.parse(userData().followed_tags).reduce(function (
```

Comment:
Okay, the naming we have for tag collections really needs work 😅 I always have to go back to the model to be sure whether this particular "followed" collection filters out tags with negative points or not

Reply:
:+1: Naming here has been a real challenge (see also: `antitags`).

Related potential future follow-up: the double-parsing in `user-data` _always_ feels weird to me and I think we could probably manage to only parse it once, but I decided it was beyond the scope of work here. (Mentioning here because we _could_ take advantage of _renaming_ `followed_tags` to possibly mitigate issues with old, outdated front-end parsing and changing back-end json.)

Reply:
I totally agree with this and I think at some point we probably do want something like following tags (points >= 0) and a hidden tags (points <=0) payload in the user data cache, but I think before we do that we need to be able to consistently apply a naming convention and a reliable output of what to expect anytime that is seen.